README.md

```diff
@@ -9,14 +9,14 @@ Common classes for creating vendor specific database extractors.
 - [MSSQL](https://github.com/keboola/db-extractor-mssql)
 - [PgSQL](https://github.com/keboola/db-extractor-pgsql)
 - [Oracle](https://github.com/keboola/db-extractor-oracle) (private repository)
-- [Impala](https://github.com/keboola/db-extractor-impala)
+- [Impala](https://github.com/keboola/db-extractor-impala) 
 - [Firebird](https://github.com/keboola/db-extractor-firebird)
 - [DB2](https://github.com/keboola/db-extractor-db2)
 
 ## Development and running tests
 
     docker-compose build
-    docker-compose run --rm dev  # runs the tests
+    docker-compose run --rm tests  # runs the tests
```

Comment:
Could we stick with "dev" as in component-generator, etc.?

Reply:
I copied `tests` from https://github.com/keboola/db-writer-common/pull/13.  Which kind of makes sense because these are libraries rather than runable components